crates/network-scheduler/src/prometheus_metrics.rs

```diff
@@ -0,0 +1,66 @@
+use std::collections::HashMap;
+
+use prometheus_client::metrics::gauge::Gauge;
+use prometheus_client::metrics::histogram::{linear_buckets, Histogram};
+use prometheus_client::registry::Registry;
+
+use crate::scheduling_unit::UnitId;
+
+lazy_static::lazy_static! {
+    static ref WORKERS_PER_UNIT: Histogram = Histogram::new(linear_buckets(0.0, 1.0, 100));
```

Comment:
Maybe we could use a family of gauges here with unit ID as a label? That would still allow to draw a histogram, but we could also get values for particular units if needed.

Reply:
Do you think we would need it in the Prometheus metrics? It will create NUM_WORKERS * 100 lines in the `/metrics` output, and Prometheus will have to store all those values for a long time. As I understand, it's always better to stick to the minimal information in the metrics and dump everything else in the logs. If we need to investigate the issue, we can get this information from the scheduler's state

Reply:
OK, you're right. Nevermind.